### ISSUE TITLE
 Do not publish solutions until the challenge ends

### ISSUE THREAD

**Issue by abeaumont** [armfergom/tuentichallenge5#1]
Please avoid leaking solutions until the challenge ends to avoid cheating and/or spoiling. Thanks

**Comment by armfergom** [comment]
Right, I didn't notice the repo was public. Will remove it when I have the time. Thanks for the heads up.
